src/logic/__tests__/calculate.test.js

```diff
@@ -0,0 +1,31 @@
+import calculate from '../calculate';
+
+describe('calculate', () => {
+  it('Adds two numbers', () => {
+    expect(calculate({ next: '1+1' }, '+').next).toBe('2+');
+    expect(calculate({ next: '7+4' }, '=').next).toBe('11');
+    expect(calculate({ next: '-7+4' }, '-').next).toBe('-3-');
+    expect(calculate({ next: '-8+-4' }, '=').next).toBe('-12');
+  });
+
+  it('Multiplies two numbers', () => {
+    expect(calculate({ next: '1.5*2' }, '+').next).toBe('3+');
+    expect(calculate({ next: '4*7.5' }, '=').next).toBe('30');
+    expect(calculate({ next: '-7*4' }, '-').next).toBe('-28-');
+    expect(calculate({ next: '-8*-4' }, '=').next).toBe('32');
+  });
+
+  it('Divides two numbers', () => {
```

Comment:
- [ ] [REQUIRED] Please add a test when user wants to divide a number by 0 because In your APP, when I do this operation, I got an error:
![1](https://user-images.githubusercontent.com/58552748/127745116-ea8ff401-8c4a-45c3-94b8-f0c217296c55.gif)